src/components/App.jsx

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react';
+
+class App extends Component {
+  render() {
+    return (
+      <div className='app'>
+        Hello World!!
+      </div>
+    );
+  }
+};
+
+export default App;
```

Comment:
please add an empty line as last line in every file (eslint)

Reply:
@ChristianSchmidt1981 
It has the empty line but it is not showing up.